npm/src/clientfileprocessing/index.ts

```diff
@@ -43,6 +43,10 @@ export class ClientFileProcessing {
 
     if (progressBarElement) {
       progressBarElement.setAttribute("value", weightedPercent.toString())
+      progressBarElement.setAttribute(
+        "aria-valuenow",
```

Comment:
Should these also be reverted if we've removed the `aria-value*` attributes from the progress bar?

Reply:
Ah yes, I forgot 😅 .